test/e2e/README.md

```diff
@@ -2,45 +2,62 @@
 
 Document for running Velero end-to-end test suite.
 
-## Command line flags for E2E tests
+The E2E tests are validating end-to-end behavior of Velero including install, backup and restore operations. These tests take longer to complete and is not expected to be part of day-to-day developer workflow. It is for this reason that they are disabled when running unit tests. This is accomplished by running unit tests in [`short`](https://golang.org/pkg/testing/#Short) mode using the `-short` flag to `go test`.
 
-Command line flags can be set after
-```
-velerocli - the velero CLI to use
-kibishiins - the namespace to install kibishii in
-cloudplatform - the cloud platform the tests will be run against (aws, vsphere, azure)
-```
+If you previously ran unit tests using the `go test ./...` command or any of its variations, then you will now run the same command with the  `-short` flag to `go test` to accomplish the same behavior. Alternatively, you can use the [`hack/test.sh`](https://github.com/vmware-tanzu/velero/blob/main/hack/test.sh) script to run unit tests.
 
-## Running tests locally
+## Prerequisites
 
-1. From Velero repository root
+Running the E2E tests expects:
+1. A running kubernetes cluster:
+    1. With DNS and CNI installed.
+    1. Compatible with Velero- running Kubernetes v1.10 or later.
+    1. With necessary storage drivers/provisioners installed.
+1. `kubectl` installed locally.
 
-    ```
-    make test-e2e
-    ```
 
-1. From `test/e2e/` directory
+## Configuration for E2E tests
 
-    ```
-    make run
-    ```
+Below is a list of the configuration used by E2E tests.
+These configuration parameters are expected as values to the following command line flags:
```

Comment:
This sentence is a little unclear - what command uses these flags ultimately? It looks like ginkgo from the Makefile, but I want to double check it's not Kibishii.

Also, is there ever a time I'd run the command versus the make target? Why?

Reply:
The flags are not for kibishii, they are mostly for installing Velero. The make target makes running these tests easy. I can't think of when someone might run ginkgo directly instead of the make target. Are you suggesting that these can be removed? If you are not then, I think they can be removed. I was not too convinced of adding it in the first place 😄

Reply:
I think it will be useful to have these documented. Makes it easy to explain what the makefile variables are being used for.

Do you have suggestions on making this clearer?

Reply:
If we are going to rely on using `make` for running these tests (which would be my preference) then I think they can be removed. As for how to make it clearer, it might be enough to just move these descriptions to the section on `make` variables below (line 47-53), replacing the `Corresponds to ...` line in each.

Reply:
@ashish-amarnath I don't have a suggestion for making it clearer, mainly because I do not know what command, what binary these command line arguments are for. They're not `velero install` arguments. What's the whole command in which I would use them? It doesn't say here, it just lists arguments.

Reply:
Put another way - what binary are these flags associated with? I'm not seeing that in this document.

Reply:
As I'm understanding, they are not flags to any command in this context, just documentation to attempt to make it more clear.

But my reaction was, why do I need to pay attention to this, will I need to set these values somewhere, in addition to the variables for the make cmd?

These flags for the install are documented elsewhere, so we could say let's just rely on that and not bother repeating that documentation in this context. With that said, if you think it would improve the understanding of what's in the makefile, then the right place for them, imo, is in the makefile.

Reply:
That's correct, these are CLI flags to the `ginkgo run` command that is run by make. The values that are passed here are used to build the install options for installing Velero.
To that effect, I added
```
These configurations or parameters are used to generate install options for Velero for each test suite.
```